### PR TITLE
Law: transfer acceleration

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+# Description
+## Suppose two reference frames, one of which is fixed (S) and the other one is moving (S'). The movement of
+## a body stationary in moving frame S' due to the movement of the frame itself is called transfer movement.
+## The acceleration related to such movement is called transfer acceleration. It is composed of the acceleration
+## of the moving frame relative to the fixed frame, centripetal acceleration and the acceleration due to uneven
+## rotation of the moving frame. The transfer acceleration only depends on the movement of frame S' relative to
+## stationary frame S, so it would be the acceleration in S of a point stationary in S'.
+
+# Law: a_tr = a_0 + cross(w, cross(w, r)) + cross(d(w)/dt, r)
+## a_tr - vector of transfer acceleration of point X relative to fixed frame S
```
Comment (blackyblack) [inline symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py:26]: Note that point X is stationary in S'

File: symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+## a_0 - vector of acceleration of moving frame S' relative to fixed frame S
```
Comment (blackyblack) [inline symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py:27]: I suppose, this vector should be attached to non-rotating point of S'

File: symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+# Description
+## Suppose two reference frames, one of which is fixed (S) and the other one is moving (S'). The movement of
+## a body stationary in moving frame S' due to the movement of the frame itself is called transfer movement.
+## The acceleration related to such movement is called transfer acceleration. It is composed of the acceleration
+## of the moving frame relative to the fixed frame, centripetal acceleration and the acceleration due to uneven
+## rotation of the moving frame. The transfer acceleration only depends on the movement of frame S' relative to
+## stationary frame S, so it would be the acceleration in S of a point stationary in S'.
+
+# Law: a_tr = a_0 + cross(w, cross(w, r)) + cross(d(w)/dt, r)
```
Comment (blackyblack) [inline symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py:25]: Might replace `cross(w, cross(w, r))` with centripetal acceleration.

Comment (blackyblack) [inline symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py:25]: Might replace `cross(d(w)/dt, r)` with rotation acceleration.

File: symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    angular_velocity_after_: QuantityVector,
+    position_vector_: QuantityVector,
+    time_before_: Quantity,
+    time_after_: Quantity,
+    time_: Quantity,
+) -> QuantityVector:
+    time = symbols("time")
+
+    # (w - w0) / (t - t0) = (w1 - w0) / (t1 - t0)
+    angular_velocity_ = add_cartesian_vectors(
```
Comment (blackyblack) [inline symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py:78]: Better pass angular velocity function from the outside.

File: symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+## The acceleration related to such movement is called transfer acceleration. It is composed of the acceleration
+## of the moving frame relative to the fixed frame, centripetal acceleration and the acceleration due to uneven
+## rotation of the moving frame. The transfer acceleration only depends on the movement of frame S' relative to
+## stationary frame S, so its physical meaning would be that it is the acceleration in S of a point stationary in S'.
+
+# Law: a_tr = a_0 + a_centripetal + a_non_uniform_rotation
+## a_tr - vector of transfer acceleration of body B relative to fixed frame S
+## a_0 - vector of acceleration of moving frame S' relative to fixed frame S
+## a_centripetal - vector of centripetal acceleration of body B in moving frame S'
+## a_non_uniform_rotation - vector of acceleration caused by non-uniform rotation of frame S'
```
Comment (blackyblack) [inline symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py:23]: Is it acceleration of a body? Maybe it's better to name it smth like `a(t)` to indicate that it is not constant?

Comment (alesanter) [reply symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py:23]: Technically, any of them can be functions of time. And this one _can_ be constant, e.g. if $`\dot w`$ is constant.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**